.dotnet/api/OpenAI.netstandard2.0.cs

```diff
@@ -1743,37 +1743,291 @@ public static class OpenAIFilesModelFactory {
+        public static implicit operator BinaryContent(FineTuningCheckpointMetrics fineTuningCheckpointMetrics);
+    }
     public class FineTuningClient {
         protected FineTuningClient();
         public FineTuningClient(ApiKeyCredential credential, OpenAIClientOptions options);
         public FineTuningClient(ApiKeyCredential credential);
         protected internal FineTuningClient(ClientPipeline pipeline, OpenAIClientOptions options);
         public FineTuningClient(string apiKey);
         public ClientPipeline Pipeline { get; }
-        public virtual FineTuningJobOperation CreateFineTuningJob(BinaryContent content, bool waitUntilCompleted, RequestOptions options = null);
-        public virtual Task<FineTuningJobOperation> CreateFineTuningJobAsync(BinaryContent content, bool waitUntilCompleted, RequestOptions options = null);
-        public virtual ClientResult GetJob(string fineTuningJobId, RequestOptions options);
-        public virtual Task<ClientResult> GetJobAsync(string fineTuningJobId, RequestOptions options);
-        public virtual CollectionResult GetJobs(string after, int? limit, RequestOptions options);
-        public virtual AsyncCollectionResult GetJobsAsync(string after, int? limit, RequestOptions options);
-    }
-    public class FineTuningJobOperation : OperationResult {
+        public virtual FineTuningJob FineTune(BinaryContent content, bool waitUntilCompleted, RequestOptions options);
+        public virtual FineTuningJob FineTune(string baseModel, string trainingFileId, FineTuningOptions options = null, CancellationToken cancellationToken = default);
+        public virtual Task<FineTuningJob> FineTuneAsync(BinaryContent content, bool waitUntilCompleted, RequestOptions options);
+        public virtual Task<FineTuningJob> FineTuneAsync(string baseModel, string trainingFileId, FineTuningOptions options = null, CancellationToken cancellationToken = default);
+        public FineTuningJob GetJob(string JobId, CancellationToken cancellationToken = default);
+        public Task<FineTuningJob> GetJobAsync(string JobId, CancellationToken cancellationToken = default);
+        public virtual CollectionResult<FineTuningJob> ListJobs(ListJobsOptions options = null, CancellationToken cancellationToken = default);
+        public virtual AsyncCollectionResult<FineTuningJob> ListJobsAsync(ListJobsOptions options = null, CancellationToken cancellationToken = default);
+    }
+    public class FineTuningError : IJsonModel<FineTuningError>, IPersistableModel<FineTuningError> {
+        public string Code { get; }
+        public string InvalidParameter { get; }
+        public string Message { get; }
+        public static explicit operator FineTuningError(ClientResult result);
+        public static implicit operator BinaryContent(FineTuningError fineTuningError);
+    }
+    public class FineTuningEvent : IJsonModel<FineTuningEvent>, IPersistableModel<FineTuningEvent> {
+        public string Level;
```

Comment:
1. According to the spec, this could be an enum with values: "info", "warn", or "error".

2. This property represents a "log level", so I recommend renaming it to `LogLevel`.